components/create_post/create_post.jsx

```diff
@@ -395,6 +407,18 @@ export default class CreatePost extends React.Component {
             userIsOutOfOffice,
         } = this.props;
 
+        if (this.props.isTimezoneEnabled) {
+            this.props.actions.getChannelTimezones(this.props.currentChannel.id).then(
```

Comment:
No action needed just a passing comment: I was going to suggest switching this to use `await` but that would delay the modal opening on the network request so I think this makes more sense